docs/site/src/demos/bottom-navigation/LabelBottomNavigation.js

```diff
@@ -32,19 +36,19 @@ export default class LabelBottomNavigation extends Component {
         >
           <BottomNavigationButton
             label="Recents"
-            icon={<span className="material-icons">restore</span>}
+            icon={<RestoreIcon />}
```

Comment:
Wait, what? That was fixed by removing `display: block`. 

What browser is that? It's fine here in Chrome and Safari, but BrowserStack doesn't seem to want to work so I can't test FF and IE.

You sure you pulled yesterday's commits?